lesson18/index.js

```diff
@@ -88,6 +109,59 @@ const controlBtnBehavior = () => {
   nextBtn.classList.remove("is-disabled");
 }
 
+const setPagenation = (slides) => {
+  const slider = document.getElementById("js-slider");
+  const pagenation = document.createElement("div");
+  pagenation.classList.add("pagenation");
+  const pagenationWrapper = document.createElement("div");
+  pagenationWrapper.classList.add("pagenationWrapper");
+  const slideLength = slides.length;
+
+  slides.map((slide,index) => {
+    const span = document.createElement("span");
+    span.classList.add("pagenationBullet");
+    index === 0 ? span.classList.add("is-active") : span.classList.remove("is-active");
```

Comment:
IMO
I don't think there is any case to remove is-active in the initial stage of setting pagination.

```suggestion
    index === 0 && span.classList.add("is-active");
```

Reply:
[67aaa11](https://github.com/hippopo-moo/js-handson-work/pull/20/commits/67aaa11b2c863f11dc86227443f2aa006c031e18)

I thought you were right, so I fixed it!
Thank you.